packages/ipfs-core/package.json

```diff
@@ -88,19 +88,19 @@
     "it-first": "^1.0.4",
     "it-last": "^1.0.4",
     "it-pipe": "^1.1.0",
-    "libp2p": "^0.29.3",
+    "libp2p": "libp2p/js-libp2p#fix/types-with-dist",
```

Comment:
the prepare script doesnt work ?

Reply:
Was going to add it now. But there are some CID type check failing now.
So I did not try it out yet

Reply:
Added it now, but type check will fail :(

Reply:
CID errors should be fixed in the latest version

Reply:
it seems to be working in CI. But my local master branch is failing with it now